### ISSUE TITLE
ProcessWrapper failing on bleeding edge windows version

### ISSUE THREAD

**Issue by jecisc** [pharo-project/pharo-launcher#114]
I tried to re-install Pharo launcher on windows but now I'm unable to launch an image.

I'm on Windows 10 and I use this installer: pharo-launcher-bleedingEdge.msi - 2018-06-04 11:07

Here is what I get:

![image](https://user-images.githubusercontent.com/9519971/40964783-44a85cd0-68ac-11e8-841f-dab1a3bcd5c0.png)

**Comment by demarey** [comment]
@jecisc Thanks for the report.
The problem comes from the shipped DLL. If you replace it by the original one (http://leves.web.elte.hu/ProcessWrapper/ProcessWrapperPlugin.dll), it should work.
I think it is a problem related to signing the installer and the app.

**Comment by jecisc** [comment]
I did that this morning and now it works :)

**Comment by demarey** [comment]
Problem fixed.
It took me some time to figure out where the problem came from. I wrongly suspected app signing.
It was a bug introduced in ConfigurationOfProcessWrapper that is in charge of downloading the DLL. In pharo < 7.0, the file was created but nothing written into it (block not evaluated). I fixed that and Hernan published it. Also thanks to @guillep to make me realize (peer debugging :) ) that the DLL was not corrupted but of size 0.

---------
Name: ConfigurationOfProcessWrapper-HernanMoralesDurand.17
Author: HernanMoralesDurand
Time: 8 June 2018, 3:52:52.640964 pm
UUID: c4b0e4a4-872a-0d00-be64-631d029b3c53
Ancestors: ConfigurationOfProcessWrapper-HernanMoralesDurand.16

Updated for Pharo 7.x
Thanks to Christophe Demarey for fixing plugin download issue.
